### PR TITLE
Update for php 7.x

### PR DESCRIPTION
**Pull request by carusogabriel**
- Version 2.9.0;
- Indicates [return](http://php.net/manual/pt_BR/migration70.new-features.php#migration70.new-features.return-type-declarations)/[variable](http://php.net/manual/pt_BR/migration70.new-features.php#migration70.new-features.scalar-type-declarations) types for security in PHP 7;
- Remove {} when ifs, elses, foreaches and whiles only have one command line;
- Padronize ifs:
	- `if($something === false) became if(!$something)`; 
	- `if($something === true) became if($something)`;

- Arrays have[ new syntax since php 5.4](https://secure.php.net/manual/pt_BR/language.types.array.php#language.types.array.syntax.array-func), so I also added in this PR;
- Fix identancion by [PHP Standard](http://www.php-fig.org/psr/);
- Fix year in LICENSE.txt
- Set minimun version required to 7.0.0 in .json

### CODE REVIEW

**Comment by bram123** [comment]
"Remove {} when ifs, elses, foreaches and whiles only have one command line;"

Doesn't this go against the PSR2 standard?

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
         }
     ],
     "require": {
-        "php": ">=5.0.0"
+        "php": ">=7.0.0"
```
Comment (ThaDafinser) [inline composer.json]: dont do this, use ^ or ~  https://github.com/Ocramius/PackageVersions/blob/master/composer.json#L13

**Comment by carusogabriel** [comment]
No, it doesn't, but removing it, more than 100 extra lines were deleted

**Comment by bram123** [comment]
http://www.php-fig.org/psr/psr-2/#control-structures
"The body of each structure MUST be enclosed by braces."

It's for readability, and without the braces it's easier to create bugs when adding a new line below.

**Comment by carusogabriel** [comment]
Ow, I didn't see that one. Let's wait this PR be merge, and I'll fixed it. Thank you

**Comment by mimmi20** [comment]
Why dont you use the codesniffer to check the coding style?

**Comment by carusogabriel** [comment]
@mimmi20 I used my IDE netbeans for that

File: composer.json (hunk 1 above)
Comment (carusogabriel) [inline composer.json]: Thanks for that one

File: Mobile_Detect.php [hunk 2]
```diff
@@ -741,10 +733,11 @@ public function getHttpHeaders()
      *
      * @return string|null The value of the header.
      */
-    public function getHttpHeader($header)
+    public function getHttpHeader(string $header)
     {
         // are we using PHP-flavored headers?
-        if (strpos($header, '_') === false) {
+        if (!strpos($header, '_'))
```
Comment (leonrenkema) [inline Mobile_Detect.php]: Please read the warning at http://php.net/strpos === should be used here.

**Comment by leonrenkema** [comment]
@carusogabriel You can update the branch and then the PR will also be updated with fixes. 

I've commentend on a row where strpos was used with === false and you removed it. You should read the warning on http://php.net/strpos. $something === false and !$something are slightly different in behaviour.

**Comment by carusogabriel** [comment]
@leonrenkema I just realize I didn't do it in a branch, so I'll have to make another PR. About the comparisons, I'll revise this part, a lot of folks are complaint about that. Thanks for that one

**Pull request closed without merging**